ext/openssl/tests/openssl-get-cipher-methods.phpt

```diff
@@ -0,0 +1,29 @@
+--TEST--
+array openssl_get_cipher_methods ([ bool $aliases = false ] );
+--CREDITS--
+marcosptf - <marcosptf@yahoo.com.br>
+--SKIPIF--
+<?php 
+if (!extension_loaded("openssl")) print "skip";
+if (OPENSSL_VERSION_NUMBER < 0x10000000) die("skip Output requires OpenSSL 1.0");
+?>
+--FILE--
+<?php
+$getCipherTrue = openssl_get_cipher_methods(true);
+$getCipherFalse = openssl_get_cipher_methods(false);
+
+if ((is_array($getCipherTrue)) && (!is_null($getCipherTrue))) {
+    print("okey");
+} else {
+    print("openssl has failure to get cipher method with true argument");
```

Comment:
This should be 'has failed' to get, and I think the end should be something like 'with the $aliases argument set to true/false'